libp2p/identity/identify/protocol.py

```diff
@@ -15,44 +18,89 @@
 from libp2p.network.stream.exceptions import (
     StreamClosed,
 )
+from libp2p.utils import (
+    get_agent_version,
+)
 
 from .pb.identify_pb2 import (
     Identify,
 )
 
+# Not sure I can do this or I break a pattern
+# logger = logging.getLogger("libp2p.identity.identify")
+logger = logging.getLogger(__name__)
```

Comment:
From the rest of the lib, it appears that 

```python
logger = logging.getLogger("libp2p.identity.identify.protocol")
```

is what would work.

Reply:
The logger name was without `.protocol` before. 
I'll use logging.getLogger(__name__) so don't have to bother with file name change.
```logging.getLogger(__name__) == logging.getLogger("libp2p.identity.identify.protocol")```